adminSiteClient/GdocsBreadcrumbsInput.tsx

```diff
@@ -0,0 +1,116 @@
+import { faPlus, faTrash } from "@fortawesome/free-solid-svg-icons"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import {
+    BreadcrumbItem,
+    OwidGdocErrorMessage,
+    OwidGdocInterface,
+} from "@ourworldindata/utils"
+import { Button, Col, Input, Row } from "antd"
+import React from "react"
+
+export const BreadcrumbLine = ({
+    item,
+    setItem,
+    removeItem,
+    urlFieldDisabled,
+}: {
+    item: BreadcrumbItem
+    setItem: (item: BreadcrumbItem) => void
+    removeItem: () => void
+    urlFieldDisabled?: boolean
+}) => {
+    return (
+        <div className="my-2">
+            <Row gutter={8}>
+                <Col span={11}>
+                    <Input
+                        addonBefore="URL"
+                        value={
+                            urlFieldDisabled
+                                ? "The last breadcrumb isn't clickable"
+                                : item.href
+                        }
+                        onChange={(e) =>
+                            setItem({ ...item, href: e.target.value })
+                        }
+                        disabled={urlFieldDisabled}
+                    />
+                </Col>
+                <Col span={11}>
+                    <Input
+                        addonBefore="Label"
+                        value={item.label}
+                        onChange={(e) =>
+                            setItem({ ...item, label: e.target.value })
+                        }
+                    />
+                </Col>
+                <Col span={2}>
+                    <Button danger onClick={removeItem}>
+                        <FontAwesomeIcon icon={faTrash} />
+                    </Button>
+                </Col>
+            </Row>
+        </div>
+    )
+}
+
+export const GdocsBreadcrumbsInput = ({
+    gdoc,
+    setCurrentGdoc,
+    errors,
```

Comment:
Open question: Should I be using `errors` somewhere? What kind of validation would be useful?